cli/azd/pkg/azure/arm_template.go

```diff
@@ -143,6 +143,7 @@ type AzdMetadata struct {
 	Type               *AzdMetadataType `json:"type,omitempty"`
 	AutoGenerateConfig *AutoGenInput    `json:"config,omitempty"`
 	DefaultValueExpr   *string          `json:"defaultValueExpr,omitempty"`
+	Default            *string          `json:"default,omitempty"`
```

Comment:
I'm wondering maybe this would be a good place to document the intended usage of the API contract